### PR TITLE
Replace User#get_by_login with a user policy

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by bgeuken** [comment]
[Replace User#get_by_login with a user policy](https://trello.com/c/fXulEBUu/718-replace-usergetbylogin-with-a-user-policy)

**Comment by bgeuken** [comment]
@hennevogel @ChrisBr Please review

File: src/api/app/policies/user_policy.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class UserPolicy < ApplicationPolicy
+  def update?
+    @user.can_modify_user?(@record)
+  end
+
+  def show?
+    @user == @record || @user.is_admin?
```
Comment (ChrisBr) [inline src/api/app/policies/user_policy.rb]: Isn't that the same as ``update?``?

Comment (bgeuken) [reply src/api/app/policies/user_policy.rb]: True. I could re-use the can_modify method.

**Pull request merged**